### PR TITLE
add kalign

### LINKED ISSUE #3757: new module: kalign
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by luisas**
## PR checklist

Closes #3757
- [x ] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/kalign/align/meta.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (sateeshperi) [inline modules/nf-core/kalign/align/meta.yml:1]:
```suggestion
```

**Review by sateeshperi — APPROVED** [review]
minor change requested. rest lgtm!

File: modules/nf-core/kalign/align/meta.yml (hunk 1 above)
Comment (JoseEspinosa) [inline modules/nf-core/kalign/align/meta.yml:1]:
```suggestion
```

File: modules/nf-core/kalign/align/main.nf [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::kalign3=3.3.5"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kalign3:3.3.5--hdbdd923_0':
+        'biocontainers/kalign3:3.3.5--hdbdd923_0' }"
+
+    input:
+    tuple val(meta),  path(fasta)
```
Comment (JoseEspinosa) [inline modules/nf-core/kalign/align/main.nf:11]:
```suggestion
    tuple val(meta), path(fasta)
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM Just to small things I spotted

**Pull request merged**